src/docfx/build/toc/parser/TableOfContentsParser.cs

```diff
@@ -25,10 +25,7 @@ public static (List<Error> errors, TableOfContentsModel model, List<Document> re
 
             var (errors, model) = LoadInternal(context, file, file, referencedFiles, referencedTocs);
 
-            var (error, monikers) = context.MonikerProvider.GetFileLevelMonikers(file);
-            errors.AddIfNotNull(error);
-
-            model.Metadata.Monikers = monikers;
+            model.Metadata.Monikers = model.Items.SelectMany(item => item.Monikers).Distinct(context.MonikerProvider.Comparer).ToList();
```

Comment:
Then need to guard `GetFileLevelMonikers` method that you should not call it on a TOC file because TOC monikers are calculated differently. This seems to affect line 191 and line 200 because monikers for inclusions are not calculated using the same algorithm.